devops/Makefile

```diff
@@ -393,6 +395,7 @@ deploy-api:
 		-p ATBC_LOGIN_ENDPOINT=$(ATBC_LOGIN_ENDPOINT) \
 		-p ATBC_ENDPOINT=$(ATBC_ENDPOINT) \
 		-p BYPASS_APPLICATION_SUBMIT_VALIDATIONS=$(BYPASS_APPLICATION_SUBMIT_VALIDATIONS) \
+		-p IS_FULLTIME_ALLOWED=${IS_FULLTIME_ALLOWED}
```

Comment:
Is there a reason this one is using a different sintax?
Should it not be `-p IS_FULLTIME_ALLOWED=$(IS_FULLTIME_ALLOWED)`?